packages/rum-core/src/domain/rumEventsCollection/view/trackInitialViewTimings.ts

```diff
@@ -150,15 +156,17 @@ export function trackFirstInputTimings(
 ) {
   const firstHidden = trackFirstHidden()
 
-  const { unsubscribe: stop } = lifeCycle.subscribe(LifeCycleEventType.PERFORMANCE_ENTRY_COLLECTED, (entry) => {
-    if (entry.entryType === 'first-input' && entry.startTime < firstHidden.timeStamp) {
-      const firstInputDelay = elapsed(entry.startTime, entry.processingStart)
-      callback({
-        // Ensure firstInputDelay to be positive, see
-        // https://bugs.chromium.org/p/chromium/issues/detail?id=1185815
-        firstInputDelay: firstInputDelay >= 0 ? firstInputDelay : (0 as Duration),
-        firstInputTime: entry.startTime as Duration,
-      })
+  const { unsubscribe: stop } = lifeCycle.subscribe(LifeCycleEventType.PERFORMANCE_ENTRIES_COLLECTED, (entries) => {
+    for (const entry of entries) {
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏same here, there can be only one "first-input" entry